docs/architecture_guide.md

```diff
@@ -12,6 +12,17 @@ Note that the output of a binarized layer is _not_ binary. Instead the output is
 
 When viewing binarization as an activation function just like ReLU, one may be inclined to binarize the outgoing activations rather than the incoming activations. However, if the network contains batch normalization layers or residual connections, this may result in unintentional non-binary operations. Therefore we have opted for an `input_quantizer` rather than an `activation_quantizer`.
 
+A typical binarized layer looks something like:
+
+```python
+x_out = lq.layers.QuantDense(
```

Comment:
I think we should either `import  larq as lq` somewhere or consistently use `larq` instead of `lq`.